gradle/scripts/lib/scala.gradle

```diff
@@ -76,7 +76,7 @@ configure(scala212) {
 
 configure(scala213) {
     dependencies {
-        implementation 'org.scala-lang:scala-library:2.13.9'
+        implementation 'org.scala-lang:scala-library:2.13.10'
```

Comment:
Sent a PR for this change.
https://github.com/line/gradle-scripts/pull/140

Reply:
I am considering adding a lookup method to find a matched version based on a regular expression. For example: `findManagedVersion('^org\.scala-lang:scala-library:2\.13\.[0-9]+$')`. It is annoying to update this version every time.